### DISCUSSION TITLE
Use Supabase DB

### DISCUSSION THREAD

**Discussion by robinbdru** [extension-js/extension.js#22] [General]
Hi guys,

I would like to be able to use Supabase for a login system but also use the database, is this possible with Extension Create?
https://supabase.com/

Thank you!

**Comment by cezaraugusto** [comment]
hi @robinbdru! There isn't an abstraction for that, but once the CLI gets more stable and better support js frameworks/browsers, I will definitely consider creating/recommending some sort of auth library for extensions. 

I created an issue for that: https://github.com/cezaraugusto/extension.js/issues/58. Feel free to share the tooling you use/would like to use for that, I'd love to hear.
